src/components/Toolbar.tsx

```diff
@@ -195,12 +196,17 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
                 <div
                   key={name}
                   id={id}
+                  style={pressingToolbarId === id ? { paddingTop: '10px', transition: 'padding 0.2s ease 0s' } : {}}
```

Comment:
If `transition` is removed when `pressingToolbarId` is no longer equal to `id`, won't that remove the return animation? I think you want `transition` to always be set, and only remove `paddingTop`. Unless I am mistaken and it already transitions both directions even when `transition` is removed.

Reply:
Yes, you are right! It will not apply animation for returning. I fixed that now.